service/history/timer_queue_active_task_executor.go

```diff
@@ -310,6 +277,66 @@ Loop:
 	return t.updateWorkflowExecution(ctx, weContext, mutableState, scheduleWorkflowTask)
 }
 
+func (t *timerQueueActiveTaskExecutor) processTimeoutTask(
+	mutableState workflow.MutableState,
+	timerSequenceID workflow.TimerSequenceID,
+	ai *persistencespb.ActivityInfo,
+	updateMutableState *bool,
+	scheduleWorkflowTask *bool,
+) (retError error) {
```

Comment:
And this shouldn't be named return value:
```suggestion
) error {
```

Reply:
removed